### ISSUE TITLE
Compatibility with NEEK2O

### ISSUE THREAD

**Issue by hardlycode** [FIX94/Nintendont#853]
Trying to launch Nintendont on USB:/APPS/ on an emulated NAND running via Neek2o causes Nintendont to lock at "Loading IOS58" consistently.

Is Nintendont compatible with Neek2o?

If it is not compatible, could you please add that information to "What Nintendont will never support:"

Thank you for your time.

**Comment by hardlycode** [comment]
Just to give a little more background:

I've read that UNEEK is the safest way to mod a Wii by not using the internal NAND and running everything from an external USB drive (in this case, a 32gb USB 3.0 Flash Drive).

My goal is to be able to run Wii and Gamecube backups completely off of USB, without the need for an SD Card and without the need for the original discs.

I've modded my original nand with the following before making a backup to use for Neek2o:
LetterBomb
HBC
Bootmii
Priiloader
CIOS Updates (d2x-cios-installer-3.1-v6-v7-v8-v10-wii, specifically V10 Beta 2 from https://wii.guide/cios)

I am unable to load Nintendont through the HBC because it does not function properly on the nand. Therefore I am using a generic USB Loader channel to get to WiiFlow, and then running Nintendont through the file explorer there.

I've managed to get NEEK2O to boot off USB and run Wii images, both WBFS and unpacked, with no problem. I've also managed to get Nintendont to run from the SD Card and run GameCube images successfully. However, attempting to load Nintendont via UNEEK results in a console freeze and requires a hard reset.

I also need support for non-GameCube controllers. Specifically, the Wii Pro Controller, which doesn't appear to be supported by the normal GameCube loader anyway.

I cannot get GameCube backups to load via NEEK2O only. The discs recognize in the Disc Channel, however, loading them causes the console to reboot out of NEEK mode.

So, again, my question is: Is Nintendont capable of running in a Neek2o environment?
